rdflib/query.py

```diff
@@ -195,10 +195,11 @@ def _set_bindings(self, b):
         _get_bindings, _set_bindings, doc="a list of variable bindings as dicts")
 
     @staticmethod
-    def parse(source, format='xml', **kwargs):
+    def parse(source=None, format=None, content_type=None, **kwargs):
```

Comment:
hmm, what happens if `source is None`?

i'd somehow find the following slightly clearer (if it's what is intended)
```python
def parse(source, format='xml', content_type=None, **kwargs):
    parser = plugin.get(content_type or format, ResultParser()
    return parser.parse(source, content_type=content_type, **kwargs)
```